### DISCUSSION TITLE
FSR sensor as trigger pads

### DISCUSSION THREAD

**Discussion by ojungleboy** [corrados/edrumulus#78] [Ideas]
Does anyone has thought to use Edrumulus instead of piezoelectric but with fsr?

I know the fundamentals of piezoelectric and fsr are different.

Is there any previous work on this?

Thanks

**Comment by corrados** [comment]
I haven't seen edrum pad designs using an FSR sensor. Why do you think it's better than a piezo sensor?

**Reply by ojungleboy** [reply]
It is most common than we think. Roland SPD uses force resistor, keyboard individual keys, akai mpc triggers. All of then are fsr.

I dont really know if they are better than piezo or who said that. I just know that they well suited to share a common mount structure. They are less suceptable to mechanical cross talk since they measure pressure on a surface and piezo vibration of the surface.

I want to place multiple sensors in a exterior of a box and avoid crosstalk such as zendrum

**Reply by corrados** [reply]
Sounds interesting. I think you have to modify the analog front end to support this type of sensor but I assume that the digital signal processing can still be the same.
